src/common/buffer.cc

```diff
@@ -1175,6 +1175,8 @@ static ceph::spinlock debug_lock;
     std::unique_ptr<buffer::ptr_node, buffer::ptr_node::disposer> nb)
   {
     unsigned pos = 0;
+    int mempool = _buffers.front().get_mempool();
+    nb->reassign_to_mempool(mempool);
```

Comment:
Yeah, lacking this reassignment seems to be actually a bug. Could you please create a ticket on the tracker and put `Fixes` into the commit message?

Reply:
Will do.